tests/earth.spec.js

```diff
@@ -0,0 +1,322 @@
+          id: 'LC8_L1T_TOA/LC81270592016009LGN00',
+        },
+        {
+          date: '2016-01-25T03:29:04',
+          id: 'LC8_L1T_TOA/LC81270592016025LGN00',
+        },
+        {
+          date: '2016-02-10T03:28:59',
+          id: 'LC8_L1T_TOA/LC81270592016041LGN00',
+        },
+        {
+          date: '2016-02-26T03:28:53',
+          id: 'LC8_L1T_TOA/LC81270592016057LGN00',
+        },
+        {
+          date: '2016-03-13T03:28:50',
+          id: 'LC8_L1T_TOA/LC81270592016073LGN00',
+        },
+        {
+          date: '2016-03-29T03:28:40',
+          id: 'LC8_L1T_TOA/LC81270592016089LGN00',
+        },
+        {
+          date: '2016-04-14T03:28:35',
+          id: 'LC8_L1T_TOA/LC81270592016105LGN00',
+        },
+        {
+          date: '2016-04-30T03:28:35',
+          id: 'LC8_L1T_TOA/LC81270592016121LGN00',
+        },
+        {
+          date: '2016-06-01T03:28:39',
+          id: 'LC8_L1T_TOA/LC81270592016153LGN00',
+        },
+        {
+          date: '2016-07-03T03:28:51',
+          id: 'LC8_L1T_TOA/LC81270592016185LGN00',
+        },
+        {
+          date: '2016-07-19T03:28:58',
+          id: 'LC8_L1T_TOA/LC81270592016201LGN00',
+        },
+        {
+          date: '2016-08-04T03:29:01',
+          id: 'LC8_L1T_TOA/LC81270592016217LGN00',
+        },
+        {
+          date: '2016-08-20T03:29:06',
+          id: 'LC8_L1T_TOA/LC81270592016233LGN00',
+        },
+        {
+          date: '2016-09-05T03:29:12',
+          id: 'LC8_L1T_TOA/LC81270592016249LGN00',
+        },
+        {
+          date: '2016-09-21T03:29:13',
+          id: 'LC8_L1T_TOA/LC81270592016265LGN00',
+        },
+        {
+          date: '2016-10-23T03:29:20',
+          id: 'LC8_L1T_TOA/LC81270592016297LGN00',
+        },
+        {
+          date: '2016-11-08T03:29:19',
+          id: 'LC8_L1T_TOA/LC81270592016313LGN00',
+        },
+        {
+          date: '2016-11-24T03:29:20',
+          id: 'LC8_L1T_TOA/LC81270592016329LGN00',
+        },
+        {
+          date: '2016-12-10T03:29:17',
+          id: 'LC8_L1T_TOA/LC81270592016345LGN00',
+        },
+        {
+          date: '2017-01-11T03:29:10',
+          id: 'LC8_L1T_TOA/LC81270592017011LGN00',
+        },
+        {
+          date: '2017-01-27T03:29:04',
+          id: 'LC8_L1T_TOA/LC81270592017027LGN00',
+        },
+        {
+          date: '2017-02-12T03:28:56',
+          id: 'LC8_L1T_TOA/LC81270592017043LGN00',
+        },
+        {
+          date: '2017-02-28T03:28:51',
+          id: 'LC8_L1T_TOA/LC81270592017059LGN00',
+        },
+        {
+          date: '2017-03-16T03:28:42',
+          id: 'LC8_L1T_TOA/LC81270592017075LGN00',
+        },
+        {
+          date: '2017-04-01T03:28:34',
+          id: 'LC8_L1T_TOA/LC81270592017091LGN00',
+        },
+      ],
+    };
+    Earth.assets({
+      lon: 100.75,
+      lat: 1.5,
+      begin: '2014-02-01',
+    }).then((data) => assert.deepEqual(data, earthAssetObj));
```

Comment:
this won't always be true unless you specify an end date. The default end date is today